.github/scripts/test-project.sh

```diff
@@ -67,6 +67,28 @@ fi
 echo "$ cd $dir/$project"
 cd "$dir/$project"
 
+# if FORCE_PRISMA_CLIENT_CUSTOM_OUTPUT is set, we execute commands that will turn the project into a custom output project
```

Comment:
This would have been worth a comment on _how_ we do that, took me a bit to understand the commands below.

Reply:
Here's a summary of what it does

```sh
○ find is used to list files.
  ◆ -name "package.json" stipulates that we search for files named package.json.
  ◆ ! -path "*/node_modules/*" stipulates that we exclude all files in the node_modules folder.
  ◆ -print0 specifies that we want to print the list of files separated by the null character.
○ xargs is used to execute a command for each item in a list.
  ◆ -0 specifies that the list is separated by the null character.
  ◆ -r specifies that we only execute the command if the list is not empty.
○ sed is used to search and replace text in files.
  ◆ -i specifies that we want to replace the text in the file.
  ◆ 's/"dependencies": {/&\n"db": "link:.\/prisma\/client",/g' is the search and replace pattern.
    ◇ s means we want to search and replace.
    ◇ "dependencies": { is the text we want to search for.
    ◇ & means we want to replace it with itself.
    ◇ \n means we want to insert a newline.
    ◇ "db": "link:.\/prisma\/client", is the text we want to insert.
    ◇ /g means we want to replace all occurrences.
 ```